Python/stream_reader_sqlrace/additional_requirements.txt

```diff
@@ -1 +1,2 @@
-pythonnet==3.0.1
```

Comment:
Why is pythonnet being removed and readded here?

Reply:
Extra line return character added, hence the diff